content/docs/other-guides/support.md

```diff
@@ -0,0 +1,183 @@
+
+* The cause falls within the technical framework that Kubeflow controls. For
+  example, the Kubeflow community may not be able to help if the problem is 
+  caused by a specific network configuration within your organization.
+* Community members can reproduce the problem.
+* The reporter of the problem can help with further diagnosis and 
+  troubleshooting.
+
+You can ask questions and make suggestions in the following places:
+
+* **Slack** for online chat and messaging. See details of Kubeflow's 
+  [Slack workspace and channels](/docs/about/community/#slack).
+
+* **Kubeflow discuss** for email-based group discussion. Join the
+  [kubeflow-discuss](https://groups.google.com/forum/#!forum/kubeflow-discuss) 
+  group.
+
+* **Kubeflow documentation** for overviews and how-to guides. In particular,
+  refer to the following documents when troubleshooting a problem:
+
+  * [Kubeflow installation and setup](/docs/started/getting-started/)
+  * [Kubeflow components](/docs/components/)
+  * [Further setup and troubleshooting](/docs/other-guides/)
+
+* **Kubeflow issue trackers** for known issues, questions, and feature requests.
+  Search the open issues to see if someone else has already logged the problem 
+  that you're encountering and learn about any workarounds to date. If no-one
+  has logged your problem, create a new issue to describe the problem.
+
+    Each Kubeflow application has its own issue tracker within the [Kubeflow
+    organization on GitHub](https://github.com/kubeflow). To get you started,
+    here are the primary issue trackers:
+
+  * [Kubeflow core](https://github.com/kubeflow/kubeflow/issues)
+  * [kfctl command-line tool](https://github.com/kubeflow/kfctl/issues)
+  * [Kustomize manifests](https://github.com/kubeflow/manifests/issues)
+  * [Kubeflow Pipelines](https://github.com/kubeflow/pipelines/issues)
+  * [Katib hyperparameter tuning](https://github.com/kubeflow/katib/issues)
+  * [Metadata](https://github.com/kubeflow/metadata/issues)
+  * [Fairing notebook SDK](https://github.com/kubeflow/fairing/issues)
+  * [TensorFlow training (TFJob)](https://github.com/kubeflow/tf-operator/issues)
+  * [PyTorch training (PyTorchJob)](https://github.com/kubeflow/pytorch-operator/issues)
+  * [KFServing](https://github.com/kubeflow/kfserving/issues)
+  * [Examples](https://github.com/kubeflow/examples/issues)
+  * [Documentation](https://github.com/kubeflow/website/issues)
+
+<a id="provider-support"></a>
+## Support from providers in the Kubeflow ecosystem
+
+The following organizations offer advice and support for Kubeflow deployments:
+
+* [Arrikto](https://www.arrikto.com). Contacts: TODO We need a point of contact.
+  Should we list some or all of the contacts that 
+  are currently in the [org list](https://github.com/kubeflow/community/blob/master/member_organizations.yaml#L42)?
+
+* TODO Are there any other organizations we should add?
+
+<a id="cloud-support"></a>
+## Support from a cloud provider
```

Comment:
Is there a more encompassing term for people providing K8s/Kubeflow solutions. For example, "OpenShift" should be listed here but I wouldn't call it a cloud.

Maybe "from a cloud or platform provider"

/cc @pdmack

Reply:
Good idea. I've changed the heading and added OpenShift.